common/changes/pcln-design-system/feat-HTL-91943-collapsible-filters_2023-08-29-19-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "pcln-design-system",
+      "comment": "remove padding from accordion header and content",
```

Comment:
Could you update the changelog comment? I think this was from the initial iteration

Reply:
yea sounds good!